README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img src="https://raw.githubusercontent.com/herbsjs/suma/master/docs/logo.png" height="220"></p>
+ <p align="center"><img src="https://raw.githubusercontent.com/herbsjs/suma/master/docs/logo.png" height="220"></p> 
 
 # Suma
```

Comment:
It is important to improve this part of the doc:
https://github.com/herbsjs/suma/#null-values